### PR TITLE
fix: UI related bugs

### LINKED ISSUE #198: [PE-D] Suggestion: GUI for help panel could match the GUI theme of the product
Perhaps you can consider getting the GUI of the help panel to match the theme! I think it would look nice :D

-------------
Labels: `type.FeatureFlaw` `severity.VeryLow`
original: YiFei2/ped#6

### LINKED ISSUE #199: [PE-D] Unable to view recipe in specific situation
Issue: 
While on the current recipe e.g. recipe 2. carbonara, editing the current recipe shows the "edited recipe" message. The user is unable to view the current recipe (recipe 1) by clicking on it and must instead click on another recipe (recipe 2), and then click back on to recipe 1. 

Steps to reproduce:
1. Have at least 2 recipes
2. Edit a recipe
3. Unable to click on the same recipe to view it and have to click on others

Clicking on "2.carbonara" does not do anything.

-------------
Labels: `type.FeatureFlaw` `severity.Low`
original: joszx/ped#3

### LINKED ISSUE #200: [PE-D] unable to view recipe after deleting all but one recipe left
Issue:
After deleting all but one recipe, the main box shows "deleted recipe: name of deleted recipe" and the user is unable to select and view the last remaining recipe.

Steps to reproduce:
1. Have 2 recipes
2. Delete recipe 2, now left with 1 recipe
3. Unable to select to view the remaining recipe

Unable to select recipe 1.spag and meatball

-------------
Labels: `type.FeatureFlaw` `severity.Medium`
original: joszx/ped#4

### PR DESCRIPTION
**Pull request by Oddcorner**
Resolves #198, resolves #199, resolves #200 

This PR fixes 2 UI related bugs:
* Click to view recipe is not functioning properly when the user clicks an already selected recipe.
* Help window CSS styling is not consistent with the main application window.

### CODE REVIEW

File: src/main/java/seedu/address/ui/RecipeListPanel.java [hunk 1]
```diff
@@ -39,6 +47,18 @@ public RecipeListPanel(ObservableList<Recipe> recipeList) {
...
+     * Sets internal selectedRecipeProperty to the selected recipe in the list.
+     */
+    private void setRecipe() {
+        Recipe recipe = selectedRecipeProperty().get();
+        if (recipe != null) {
+            // workaround to trigger selection change
+            recipeListView.getSelectionModel().select(null);
+            recipeListView.getSelectionModel().select(recipe);
+        }
+    }
```
Comment (yusufaine) [inline src/main/java/seedu/address/ui/RecipeListPanel.java:60]: HAHAHA such a cheeky way to do it but it works!

**Review by yusufaine — APPROVED** [review]
Looks good to me!

Can confirm the select issue has been rectified and the help command is using the same theme. 🔥

**Review by blaketingyu — APPROVED** [review]
LGTM!!

**Review by gigituang — APPROVED** [review]
LGTM!! 🌮

**Review by aidanaden — APPROVED** [review]
LGTM! thank god this guy caught this

**Pull request merged**